internal/ast/utilities.go

```diff
@@ -2104,6 +2104,9 @@ func TryGetTextOfPropertyName(name *Node) (string, bool) {
 }
 
 // True if node is of a JSDoc kind that may contain comment text.
+// NOTE: while this is a faithful port of Strada's implementation,
```

Comment:
Should Comments just check this and return nil or something?

Reply:
Oh, I got totally confused in my original answer, I thought Jake's comment was about a different function. But I agree we should rename it, I just don't know to what. In Strada this function even has a comment saying "True if node is of a kind that may contain comment text", so maybe this function was wrong in Strada too? I'm not sure. Maybe we should rename to `DoNotUseVeryBadIsJSDocCommentContainingNode` 😅

Reply:
I renamed it and also moved it to the astnav package, the only place where it's used. Couldn't think of a good name because I don't really know what the function is supposed to do.